### ISSUE TITLE
learning rate optimization

### ISSUE THREAD

**Issue by MichaelD7911** [zcablii/SARDet_100K#41]
I've looked through `schedule_1x.py` for learning rate schedule.

It looks very similar to on of MMengine default suggestions (https://mmengine.readthedocs.io/en/latest/tutorials/param_scheduler.html):

```
param_scheduler = [
    # Linear learning rate warm-up scheduler
    dict(type='LinearLR',
         start_factor=0.001,
         by_epoch=False,  # Updated by iterations
         begin=0,
         end=50),  # Warm up for the first 50 iterations
    # The main LRScheduler
    dict(type='MultiStepLR',
         by_epoch=True,  # Updated by epochs
         milestones=[8, 11],
         gamma=0.1)
]
```

Did you train to change it ?
Does it obtains best results, from what you tried ?

**Comment by zcablii** [comment]
No, I kept the default COCO dataset settings, as SARDet-100K shares many similarities with COCO. We found that the AdamW optimizer outperforms SGD, so we used AdamW. After experimenting with different learning rates, we found that 0.0001 is stable and provides good performance.
